ui-modules/blueprint-composer/app/components/spec-editor/spec-editor.template.html

```diff
@@ -323,7 +323,7 @@ <h4>No matching parameters</h4>
           <div ng-if="model.miscData.get('config').length === 0">
             <h4>No configuration</h4>
             <p class="buttons">
-                <button class="btn btn-sm btn-success" ng-click="specEditor.addConfigKey(state.config.search)" ng-if="state.config.search">
+                <button class="btn btn-sm btn-success" ng-click="specEditor.addConfigKey(state.config.search)" ng-if="state.config.search && state.config.search!=''">
```

Comment:
Don't believe this is needed as it adds complexity and  empty `''` strings evaluate to false in `if` statements anyways. https://stackoverflow.com/questions/27380000/angularjs-ng-if-check-string-empty-value

Reply:
that's what i thought @jathanasiou but for some reason without this i was seeing an `Add ''` button and adding this seemed to fix it!  not sure what is going on.

Reply:
Sounds like an angular 1 bug. I saw some mentions online of how `ng-if` evaluates expressions so maybe we're victims of that.